change/@fluentui-react-4726e359-06af-4ddf-baa2-635fc6b47b59.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33267&commitId=9495f5764f7fbaa733ea7068fcba5bd1dba68921&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.No callout width specified.chromium.png|2319| Changed|
|Callout.No beak.chromium.png|2306| Changed|
|Callout.Bottom left edge.chromium.png|2309| Changed|
|Callout.Bottom auto edge.chromium.png|2309| Changed|
|Callout.Right top edge.chromium.png|1127| Changed|
|Callout.Bottom Left Edge RTL.chromium.png|2178| Changed|

</details>


  <details><summary><b>react-charting-VerticalBarChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis.chromium.png|4| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>